libraries/azure_generic_resource.rb

```diff
@@ -144,4 +144,23 @@ def resource_group
   def failed_resource?
     @failed_resource ||= false
   end
+
+  # Create properties on a resource acquired via additional API call in a static method.
+  # @param opts [Hash]
+  #   property_name [string] The name of the property.
+  #   property_endpoint [string] The URI of the properties.
+  #     E.g., '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/
+  #     Microsoft.Sql/servers/{serverName}/firewallRules'.
+  #   api_version [string] The api version of the endpoint (default - latest).
+  def create_additional_properties(opts = {})
```

Comment:
Thoughts on changing the method to `additional_resource_properties` ?

Reply:
We can update 👍

Reply:
Updated.